Makefile

```diff
@@ -1057,9 +1057,12 @@ heat_kuttl_run: ## runs kuttl tests for the heat operator, assumes that everythi
 	INSTALL_YAMLS=${INSTALL_YAMLS} kubectl-kuttl test --config ${HEAT_KUTTL_CONF} ${HEAT_KUTTL_DIR}
 
 .PHONY: heat_kuttl
-heat_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy heat heat_deploy_prep heat_deploy  ## runs kuttl tests for the heat operator. Installs openstack crds and keystone operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+heat_kuttl: namespace input openstack_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy rabbitmq rabbitmq_deploy infra heat heat_deploy_prep heat_deploy  ## runs kuttl tests for the heat operator. Installs openstack crds and keystone operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
```

Comment:
Last thing I promise :) We should remove the `heat_deploy` as dependency, as that is done as part of the kuttl test run

Reply:
trying to push the change but there is a problem.

Reply:
finally working